### PR TITLE
add pcap file export

### PR DESCRIPTION
**Pull request by appl3s**
: )

### CODE REVIEW

File: ios/pcap/pcap.go [hunk 1]
```diff
@@ -1,18 +1,55 @@
...
+	HdrSize        int `struc:"uint32,big"`
+	Xxx            int `struc:"uint8,big"`
+	PacketSize     int `struc:"uint32,big"`
+	Flag1          int `struc:"uint32,big"`
+	Flag2          int `struc:"uint32,big"`
+	OffsetToIPData int `struc:"uint32,big"`
+	Zero           int `struc:"uint32,big"`
+}
+
 func Start(device ios.DeviceEntry) error {
```
Comment (danielpaulus) [inline ios/pcap/pcap.go:42]: Could you please add a comment explaining what the function does?

File: ios/pcap/pcap.go [hunk 2]
```diff
@@ -1,18 +1,55 @@
...
 	intf, err := ios.ConnectToService(device, "com.apple.pcapd")
 	if err != nil {
 		return err
 	}
 	plistCodec := ios.NewPlistCodec()
+	f, err := createPcap(fmt.Sprintf("dump-%d.pcap",time.Now().Unix()))
```
Comment (danielpaulus) [inline ios/pcap/pcap.go:48]: please add a log Info printing the filename using logrus.

File: ios/pcap/pcap.go [hunk 3]
```diff
@@ -31,3 +76,65 @@ func fromBytes(data []byte) ([]byte, error) {
...
+		return nil, err
+	}
+	f.Write([]byte{
+		0xd4, 0xc3, 0xb2, 0xa1, 0x02, 0x00, 0x04, 0x00,
+		0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+		0xff, 0xff, 0x00, 0x00, 0x01, 0x00, 0x00, 0x00})
+	return f, nil
+}
+
+type PcaprecHdrS struct {
```
Comment (danielpaulus) [inline ios/pcap/pcap.go:92]: can you add a comment where to find the header files for these maybe?

File: ios/pcap/pcap.go [hunk 4]
```diff
@@ -31,3 +76,65 @@ func fromBytes(data []byte) ([]byte, error) {
...
+
+	interfacetype := make([]byte, pph.HdrSize-25)
+	cnt, err := preader.Read(interfacetype)
+	if err != nil {
+		return []byte{}, err
+	}
+	if cnt != pph.HdrSize-25 {
+		return []byte{}, errors.New("invalid HdrSize")
+	}
+	fmt.Println(interfacetype)
```
Comment (danielpaulus) [inline ios/pcap/pcap.go:130]: please remove or use logrus to log.Info this :-)

**Review by danielpaulus — CHANGES_REQUESTED** [review]
This is really cool! Thank you for the work and making go-iOS better. 
I have just a few nitpicky change requests, I would really appreciate if you found the time to make them.

**Comment by appl3s** [comment]
👌

**Pull request closed without merging**